samples/chart-docs/TableChartSamples.jsx

```diff
@@ -152,6 +152,9 @@ export default class TableChartSamples extends React.Component {
                                                         <strong>type</strong> - Type of the chart that need to be
                                                         visualized in this case &qoute;table&qoute;
                                                     </li>
+                                                    <li>
+                                                        <strong>deduplicationColumn</strong> - To stop duplication of data
```

Comment:
This documentation doesn't explain what is `deduplicationColumn` and how it is going to be used. IMO, `deduplicationColumn` is not a very intuitive name, how about `uniqueProperty:  Name of a property in a data object that has to carry an unique value. This will be used to eliminate duplicates in an array of data`

Reply:
@CharukaK Please attend to above before releasing to NPM

Reply:
shall we add *uniquePropertyColumn* as the key?

Reply:
+1